contract/src/main/kotlin/com/r3/corda/sdk/token/contracts/EvolvableTokenContract.kt

```diff
@@ -17,10 +17,15 @@ import net.corda.core.transactions.LedgerTransaction
 abstract class EvolvableTokenContract : Contract {
 
     override fun verify(tx: LedgerTransaction) {
+
+        // Only allow a single command
+        require(tx.commandsOfType<EvolvableTokenTypeCommand>().size == 1) { "Evolvable token transactions support exactly one command only." }
+
+        // Dispatch based on command
         val command = tx.commands.requireSingleCommand<EvolvableTokenTypeCommand>()
         when (command.value) {
-            is Create -> handleCreate(tx)
-            is Update -> handleUpdate(tx)
+            is Create -> verifyCreate(tx)
```

Comment:
We should come up with some standardised naming here - not sure what I called them in the other contracts.